.github/workflows/main.yml

```diff
@@ -37,3 +37,25 @@ jobs:
         run: |
           echo $GITHUB_WORKSPACE
           docker run --privileged -e CI=true -t -v $GITHUB_WORKSPACE:/github/workspace docker.pkg.github.com/fwupd/fwupd/fwupd-${{matrix.os}}:latest
+
+  fuzzing:
+    runs-on: ubuntu-latest
```

Comment:
Do you want this only running on master perhaps?

Reply:
I wanted it on each branch so that the plugin submitter would have to fix fuzzing issues *before* we merged to master and it became *our* problem.

Reply:
So the expectation is that plugin submitter integrates the plugin into oss-fuzz as _part_ of the submission?

Reply:
I think for anything that's doing more than _safe() stuff in the FuXxxxxFirmware we should expect the author to implement ->build() and ->write() so that we (they?) can construct minimal fuzzing targets.

Reply:
I guess that should be documented in https://fwupd.org/lvfs/docs/consulting/info as well?

Reply:
I think adding it to https://github.com/fwupd/fwupd/wiki/New-plugin-checklist is probably better -- maybe something like:

  [X] Implement ->write() and include a fuzzer testcase in src/fuzzing/firmware for any custom FuFirmware subclass

Reply:
Oh yeah.  Sounds good

Reply:
Done.